benchmarks/run_benchmarks.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+benchmark_execution_timeout=10800 # 3 hours
+
+read -p "Enter your aws access key: "  aws_access_key
```

Comment:
It would be great if the following PR implements the jenkins file which would call this script. That jenkins pipeline could run for example every week or something and would periodically run benchmarks on AWS.